pkg/init/cmd/service/cmd.go

```diff
@@ -150,6 +151,35 @@ func stop(ctx context.Context, service, sock, basePath string) (string, uint32,
 	return id, pid, "", nil
 }
 
+func newLogDestination(name string) string {
+	path := filepath.Join("/var/log", name+".log")
+
+	if loggingEnabled() {
+		if err := syscall.Mkfifo(path, 0600); err != nil {
```

Comment:
If I read this code right we either have `/var/log/<name>.log` being a text file or a FIFO. It might be better to have the FIFO in in `/var/run` to keep `/var/log` for text files (optionally rotated later)